rdmo/projects/views/project_create.py

```diff
@@ -45,6 +43,7 @@ def form_valid(self, form):
 
         # add all tasks to project
         tasks = Task.objects.filter_current_site() \
+                            .filter_catalog(self.object.catalog) \
```

Comment:
I think this was missing here, but I am not super sure, maybe @MyPyDavid or @m6121 can check.

Reply:
Good catch! I also think this should be here.

Reply:
but now still all the tasks are added to a project even when posting:
```json
{
"title": "Projekt XY 0 view 1 task 1 ",
"description": "nur mal so",
"catalog": 1,
"views": [1],
"tasks": [1]
}
```
Is this the desired behaviour? 
`Tasks` behave differently from the `Views` because its a through model with `Issue`??

Reply:
Yes, exactly. What also does not work ist posting `views: []` to not get any views. We might address this in the future, but I think right now this is enough to fix #593.